SportsHub/SportsHub.Api/Controllers/ArticleController.cs

```diff
@@ -0,0 +1,36 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using SportsHub.AppService.Authentication.Models.DTOs;
+using SportsHub.AppService.Services;
+using System.Text.Json;
+
+namespace SportsHub.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ArticleController : ControllerBase
+    {
+        private readonly IArticleService service;
+        private readonly IMapper mapper;
+
+        public ArticleController(IArticleService service, IMapper mapper)
+        {
+            this.service = service;
+            this.mapper = mapper;
```

Comment:
rename private fields, start should be with _, and do it everywhere! remove "this". I think I with Mykola told about it, already.

Reply:
Fixed